app/assets/javascripts/student_profile/ProfilePdfDialog.js

```diff
@@ -134,18 +145,22 @@ export default class ProfilePdfDialog extends React.Component {
                 datepicker: styles.datepickerContainer,
                 input: styles.datepickerInput
               }}
-              value={this.state.filterToDate.format('MM/DD/YYYY')}
+              value={this.state.filterToDateText}
               onChange={this.onFilterToDateChanged}
               datepickerOptions={{
                 showOn: 'both',
                 dateFormat: 'mm/dd/yy',
                 minDate: undefined
               }} />
           </div>
+          <div style={{height: '2em'}}>
+            {!this.areDatesValid() && <div className="PdfDialogue-warning" style={styles.invalidDate}>Choose a valid date</div>}
```

Comment:
It looks like you defined a `areDatesValid` variable up above, so you could use that here too.

Also, I don't see where `styles.invalidDate`is defined, should that be added with something like orange or red text, or maybe removed?